src/_guides/language/effective-dart/documentation.md

```diff
@@ -257,8 +257,10 @@ int weekday;
 int get checkedCount => ...
 {% endprettify %}
 
-Avoid having a doc comment on both the setter and the getter, as DartDoc will show
-only one (the one on the getter.)
+When you have a corresponding getter and setter with the same name, document
+only one of them. Dartdoc will merge them together and document them as if they
+are a single field. When it does that, if both the getter and setter have doc
+comments, it discards the setter's.
```

Comment:
```suggestion
If a property has both a getter and a setter, then create a doc comment for
only one of them. Dartdoc treats the getter and setter like a single field,
and if both the getter and the setter have doc comments, then
dartdoc discards the setter's doc comments.
```

Reply:
My suggestions are an attempt to remove some possible ambiguity, and we (conforming to Google's style guide) also prefer present tense.